src/objects/zcl_abapgit_object_tabl.clas.abap

```diff
@@ -661,7 +661,8 @@ CLASS zcl_abapgit_object_tabl IMPLEMENTATION.
           lv_refs      TYPE abap_bool,
           ls_extras    TYPE ty_tabl_extras.
 
-    FIELD-SYMBOLS: <ls_dd03p> TYPE dd03p.
+    FIELD-SYMBOLS: <ls_dd03p> TYPE dd03p,
+                   <lg_roworcolst> TYPE any.
```

Comment:
remove changes in this class in your fork, since they are part of a different pull request.